### DISCUSSION TITLE
Power 1 toggle time limit?

### DISCUSSION THREAD

**Discussion by uxhamby** [arendst/Tasmota#19395] [Q&A]
HI,

I set up an 8266 with a time limit on the Power1 toggle.

As I recall there was a command line 'setoptionX' that did this for me but I can no longer remember the details on how I did it.

Could someone pls help me with a memory jog on this?

Thanks,

Brian H.
Uxbridge Ont

**Comment by barbudor** [comment]
Not sure what you mean with toggle time limit

There is a PulseTime command which allows to automatically turn off power after a certain time (or when used with `PowerOnState 5` does the opposite- turn off for a limited time)

Is taht what you are looking for ?

If not, please elaborate what is your goal / wanted behavior

**Reply by uxhamby** [reply]
PulseTime. Yes, that seems to be it. Thanks!

Brian H.
